packages/devextreme/testing/tests/DevExpress.viz.sparklines/sparkline.tests.js

```diff
@@ -1433,6 +1433,23 @@ QUnit.begin(function() {
         ]);
     });
 
+    QUnit.test('winloss sparkline lossColor should change', function(assert) {
```

Comment:
'The main issue is that changes are not reflected after a runtime change. Please adjust the title accordingly' - the same goes for the test name